test/cpp/ext/otel/otel_test_library.h

```diff
@@ -59,7 +59,7 @@ class MockMetricReader : public opentelemetry::sdk::metrics::MetricReader {
 };
 
 class OpenTelemetryPluginEnd2EndTest : public ::testing::Test {
- protected:
+ public:
```

Comment:
Why does this need to be public instead of protected?  I don't think this should be used by anything but subclasses.

Is this because of the standalone `ConfigureOTBuilder()` function you have in the .cc file?  If so, I suggest making that a static method on the class instead of a standalone function.

Reply:
exactly, done